lib/index.js

```diff
@@ -44,11 +45,28 @@ function qRouter(spec) {
                     const cascade = targets.some(target => {
                         if ('cond' in target) {
                             // Do any of the conditions reference the sectionId
-                            return target.cond.some(element =>
-                                String(element).includes(`.${sectionId}.`)
-                            );
+                            if (target.cond.some(element => String(element).includes('|role'))) {
```

Comment:
Could we extract a method here, with some indication of intent?

Reply:
Extracted into new function